### PR TITLE
`xml` stubs: `Any` -> `Incomplete`

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: stdlib/xml/dom/minidom.pyi [hunk 1]
```diff
@@ -87,7 +86,7 @@ class NamedNodeMap:
     def keys(self): ...
     def keysNS(self): ...
     def values(self): ...
-    def get(self, name: str, value: Any | None = ...): ...
+    def get(self, name, value: Incomplete | None = ...): ...
```
Comment (srittau) [inline stdlib/xml/dom/minidom.pyi]: Any reason you removed the `name` annotation?

Comment (AlexWaygood) [inline stdlib/xml/dom/minidom.pyi:89]:
```suggestion
    def get(self, name: str, value: Incomplete | None = ...): ...
```

**Review by srittau — APPROVED** [review]
One question, but LGTM apart from that. (Also `Incomplete` to `Any` is the "safe" direction.)

File: stdlib/xml/dom/xmlbuilder.pyi [hunk 2]
```diff
@@ -60,7 +61,7 @@ class DOMBuilder:
     def supportsFeature(self, name: str) -> bool: ...
     def canSetFeature(self, name: str, state: int) -> bool: ...
     # getFeature could return any attribute from an instance of `Options`
-    def getFeature(self, name: str) -> Any: ...
+    def getFeature(self, name: str) -> Incomplete: ...
```
Comment (AlexWaygood) [inline stdlib/xml/dom/xmlbuilder.pyi:64]:
```suggestion
    def getFeature(self, name: str): ...
```

File: stdlib/xml/dom/minidom.pyi (hunk 1 above)
Comment (AlexWaygood) [reply stdlib/xml/dom/minidom.pyi]: It was a bad merge, I already fixed it :)

File: stdlib/xml/dom/xmlbuilder.pyi [hunk 3]
```diff
@@ -60,7 +61,7 @@ class DOMBuilder:
     def supportsFeature(self, name: str) -> bool: ...
     def canSetFeature(self, name: str, state: int) -> bool: ...
     # getFeature could return any attribute from an instance of `Options`
-    def getFeature(self, name: str) -> Any: ...
+    def getFeature(self, name: str): ...
```
Comment (AlexWaygood) [inline stdlib/xml/dom/xmlbuilder.pyi:64]:
```suggestion
    def getFeature(self, name: str) -> Incomplete: ...
```

**Review by srittau — APPROVED** [review]

**Pull request merged**